character/src/sass/main.scss

```diff
@@ -4,136 +4,150 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-/* Compiles to: css/main.css */
-
-html, body {
-  margin: 0;
-  padding: 0;
+body {
   overflow: hidden;
+  cursor: default;
+  height: 100%;
+  width: 100%;
+  margin: 0;
+  font-family: sans-serif;
+  color: white;
 }
 
-/* Target ONLY cse-ui-character contained elements */
+#cse-ui-wounds {
```

Comment:
instead of copy pasting scss, add it to the styles in the camelot-unchained package.  Then you would just either import it here, or include the css on the html.